modules/installation-aws-config-yaml.adoc

```diff
@@ -177,6 +177,10 @@ endif::china[]
 ifdef::china[]
         url: https://vpce-id.ec2.cn-north-1.vpce.amazonaws.com.cn
 endif::china[]
+ifdef::secret[]
+      - name: route53 <9>
+        url:  https://route53.us-east-1.amazonaws.com
```

Comment:
Based on the [AWS doc](https://docs.aws.amazon.com/general/latest/gr/r53.html), I specified `us-east-1` as the region. Please confirm that this is valid for our implementation.